src/platform/forms-system/src/js/fields/ObjectField.jsx

```diff
@@ -226,9 +198,9 @@ class ObjectField extends React.Component {
         : idSchema.$id;
 
     const fieldContent = (
-      <div className={containerClassNames}>
+      <>
         {hasTitleOrDescription && (
-          <div className="schemaform-block-header">
+          <>
```

Comment:
Removing these `<div>`s should fix the accessibility error by preventing a `<div>` from being the top level component inside of a `<fieldset>`, which would happen when the schema had a title.